locations/spiders/normal.py

```
@@ -12,7 +12,7 @@
 class NormalSpider(Spider):
     name = "normal"
     item_attributes = {"brand": "Normal", "brand_wikidata": "Q19562429"}
-    start_urls = ["https://www.normal.no/stores?culture=en-150"]
+    start_urls = ["https://www.normalstores.com/stores?culture=en-150"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)